swarmer/src/main/kotlin/com/gojuno/swarmer/Args.kt

```diff
@@ -46,7 +46,7 @@ sealed class Commands {
                     description = "Package of the system image for this AVD (e.g.'system-images;android-25;google_apis;x86') to pass to `avdmanager create avd --package`.",
                     order = 2
             )
-            var pakage: String = "",
+            var `package`: String = "",
```

Comment:
But why? It's just internal name :)

I suspect we can have problems with kapt (if we will use it in future) with this name, because it's reserved in Java

cc @ming13, @nostra13 (holy k, Artur copy-pasted your nick!!1)